src/compiler/commandLineParser.ts

```diff
@@ -338,7 +338,10 @@ namespace ts {
         },
         {
             name: "listEmittedFiles",
-            type: "boolean"
+            type: {

```

Comment:
If you instead just make this a string which is the file path to write to (where "-" indicates stdout), then this would be perfect. It's preferable not to rely on both the working directory and a filename hardcoded into the compiler. :)